rio_stac/scripts/cli.py

```diff
@@ -0,0 +1,77 @@
+"""rio_stac.scripts.cli."""
+import json
+
+import click
+from rasterio.rio import options
+
+from rio_stac import create_stac_item
+
+
+def _cb_key_val(ctx, param, value):
+    if not value:
+        return {}
+    else:
+        out = {}
+        for pair in value:
+            if "=" not in pair:
+                raise click.BadParameter(
+                    "Invalid syntax for KEY=VAL arg: {}".format(pair)
+                )
+            else:
+                k, v = pair.split("=", 1)
+                out[k] = v
+        return out
+
+
+@click.command()
+@options.file_in_arg
+@click.option(
+    "--datetime",
+    "-d",
+    type=str,
+    help="The searchable date and time of the assets, in UTC.",
+)
+@click.option(
+    "--with-proj/--without-proj", default=True, help="Add PROJ extension and metadata."
```

Comment:
Should this be part of `--extension`? `rio stac --extension proj`?

Reply:
yeah, because I wanted `proj` to be set by default, I though it was clearer to add it as a separate option 🤷‍♂️

Reply:
Well you can default to `extensions=['proj']`, you'd just have to make it clear that the user needs to include it if setting custom extensions.